go.mod

```diff
@@ -1,19 +1,31 @@
 module github.com/nabancard/terraform-provider-kubernetes-yaml
 
-go 1.12
+go 1.13
 
 require (
+	cloud.google.com/go/bigtable v1.3.0 // indirect
 	github.com/cenkalti/backoff v2.1.1+incompatible
-	github.com/google/btree v1.0.0 // indirect
-	github.com/hashicorp/terraform v0.12.0
-	github.com/hashicorp/yamux v0.0.0-20181012175058-2f1d1f20f75d // indirect
+	github.com/dustinkirkland/golang-petname v0.0.0-20191129215211-8e5a1ed0cff0 // indirect
+	github.com/hashicorp/terraform v0.11.11 // indirect
```

Comment:
Why is Terraform module being rolled back here from `v0.12.0` to `v0.11.11` ?

Reply:
Sorry we can close this one in favor of the other which is a better and cleaner  patch.